### PR TITLE
Rullzer prevent inf loop

### PR DESCRIPTION
**Pull request by DeepDiver1975**
Follow up of #70 

screenshot from 2018-09-25 15-41-42

### CODE REVIEW

**Comment by staabm** [comment]
Did you check the test will fail without the patch?

**Comment by rullzer** [comment]
Thanks for taking over @DeepDiver1975

**Comment by DeepDiver1975** [comment]
This unit test is not hitting the code in question - I need to have a second look ....

File: tests/Sabre/Xml/ServiceTest.php [hunk 1]
```diff
@@ -155,6 +155,25 @@ function testExpectStream() {
         );
     }
 
+    function testEmptyProps() {
```
[the remark this answers is not in the document]
Comment (rullzer) [reply tests/Sabre/Xml/ServiceTest.php]: I took a look but it indeed does not seem to be triggered anymore. Maybe there was a different bug. Anyway good it is fixed.

[the remark this answers is not in the document]
Comment (DeepDiver1975) [reply tests/Sabre/Xml/ServiceTest.php]: Maybe related to an older version. Let's close this for now - let us know if the issue pops up again. THX

**Comment by DeepDiver1975** [comment]
@rullzer after having another look I can now confirm the bug - here is the fix: https://github.com/sabre-io/xml/pull/158

Maybe you find the time for some testing? THX

**Pull request closed without merging**